e220/e220.go

```diff
@@ -0,0 +1,349 @@
+//go:build tinygo
+
+// Package e220 implements a driver for the e220 LoRa module.
+//
+// Datasheet(E220-900T22S JP): https://dragon-torch.tech/wp-content/uploads/2022/08/data_sheet.pdf
+// Datasheet(E220-900T2S): https://www.ebyte.com/en/downpdf.aspx?id=1211
+// Datasheet(LLCC68): https://semtech.my.salesforce.com/sfc/p/#E0000000JelG/a/2R000000HTJR/Tem0gUxGfOZ2Qn3bUzmV2zKNQRYJ3bpobPfOQ7B.erE
+//
+// LoraWAN 1.1 Specification: https://hz137b.p3cdn1.secureserver.net/wp-content/uploads/2020/11/lorawantm_specification_-v1.1.pdf?time=1673563697
+package e220
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"machine"
+	"runtime"
+	"strings"
+	"time"
+)
+
+// targetUARTbaud represents the UART baud rate of the target board
+const (
+	TargetUARTBaud1200kbps   = 1200
+	TargetUARTBaud2400kbps   = 2400
+	TargetUARTBaud4800kbps   = 4800
+	TargetUARTBaud9600kbps   = 9600
+	TargetUARTBaud19200kbps  = 19200
+	TargetUARTBaud38400kbps  = 38400
+	TargetUARTBaud57600kbps  = 57600
+	TargetUARTBaud115200kbps = 115200
```

Comment:
Does this add any value? It seems like a round about way to say `BaudRate1200` is equal to `1200`. MAybe this should be a `Baud` type instead and these be reduced to `Baud1200`.